README.md

```diff
@@ -11,9 +11,10 @@
 
 - numpy (install by `conda`)
 - pyyaml (install by `conda`)
-- pyside2 (install by `conda`)
-- qt5-qtbase-devel (install by `yum`)
-- xorg-x11-server-Xvfb (optional, install by `yum`)
+- pyside6 (install by `conda`)
+- qt6-charts (install by `conda`)
+- qt5-qtbase-devel (install by `dnf`)
```

Comment:
Are you sure this and the next line are needed? When using conda all required system libraries get installed without the need for using dnf.

Reply:
That was from the previous ticket when we experimented the PySide2. Let us keep this for now and test the minimum dependency with PySide6 when we are ready to remove the PySide2. As you know, the QT framework is a headache (at least, a little better than LabVIEW...) Thanks!